3 - PB/Documentazione interna/Norme di Progetto/Norme di Progetto.typ

```diff
@@ -1830,50 +1830,76 @@ Per avere traccia degli elementi verificati che costituiscono il prodotto, su Gi
+
+Ogni release avvenuta con successo, in conformità con il superamento dei test e le procedure descritte nei processi di implementazione (@processo_implementazione), integrazione (@processo_integrazione) e verifica (@processo_verifica), genera una release specifica tracciata e consultabile (@tracciabilità_risultati_release) con relativo numero di release aggiornato.
+
+Ad ogni modifica del sistema devono venire aggiornati anche i documenti #man e #st, al fine di includere la documentazione relativa alle modifiche apportate.
 
-==== Rilascio dell'ultima versione del sistema
-Il repository GitHub dedicata a WMS3 si compone di un branch `dev` necessario per gli sviluppatori, in quanto fonte da cui vengono creati i feature branch necessari al conseguimento degli obiettivi di una task e destinazione dei merge a seguito delle successive pull request. Viene quindi definito un altro branch `release`, orfano e luogo in cui vengono effettuate i rilasci del sistema.
+==== Deploy su Azure
+
+Il gruppo ha definito una GitHub Action chiamata _Deploy to VPS_ (i cui job sono descritti nel file _deploy.yml_) la quale effettua il deploy del sistema, aggiornato all'ultima versione rilasciata sul branch `dev`, all'interno della macchina virtuale di Azure.
 
-I rilasci avvengono solo quanto il sistema viene ritenuto sufficientemente maturo e pronto per l'utilizzo o quando vengono:
-- aggiunte modifiche sostanziali al sistema rispetto all'ultima versione rilasciata;
-- risolti bug critici;
-- avanzamento del sistema rilasciato ad una versione più aggiornata.
-I rilasci del prodotto avvengono sempre dal branch di partenza `dev` a quello di destinazione `release`, in quanto solo il codice contenuto in `dev` soddisfa i requisiti di qualità imposti dal gruppo e permette la corretta esecuzione dei processi di implementazione (@processo_implementazione), integrazione (@processo_integrazione) e verifica (@processo_verifica).
+Questo permette la fruizione del sistema senza la necessità di dover scaricare le ultime modifiche sul proprio dispositivo e senza dover ricreare le relative immagini Docker.
 
-Ad ogni release del sistema devono essere rilasciati anche i documenti #man e #st aggiornati che documentano le novità apportate.
 
 ==== Redazione del documento #man
-Al fine di istruire gli utenti del sistema al suo corretto utilizzo, il gruppo mette a disposizione il documento #man.
 
-In questo documento viene riportato in particolare:
+Al fine di fornire agli utenti di WMS3 una guida documentale atta ad indicare il suo corretto utilizzo, il gruppo mette a disposizione il documento #man.
+
+In esso viene riportato:
+
 - installazione locale del sistema;
-- best practice, descrizione dei servizi offerti dal sistema;
-- guida all'utilizzo delle interfacce per la creazione, modifica e popolamento dell'ambiente 3D.
+
+- guida all'avvio e configurazione dell'ambiente;
+
+- descrizione dei servizi offerti dal sistema;
+
+- guida dettagliata all'utilizzo dei servizi offerti.
```

Comment:
In conseguenza al commento precedente modificherei ~~dei servizi offerti~~ -> delle funzionalità offerte